modules/gitops-configuring-argo-cd-oidc.adoc

```diff
@@ -0,0 +1,79 @@
+// Module is included in the following assemblies:
+//
+// * 
+
+[id="configuring-argo-cd-oidc_{context}"]
+= Configuring Argo CD OIDC
+
+To configure Argo CD OpenID Connect (OIDC), you must generate your client secret, encode it and add it to your Custom Resource.
+
+.Prerequisites
+
+* You have obtained your client secret. 
+
+.Procedure
+
+. Store the client secret you generated.
+
+* Encode the client secret in base64:
```

Comment:
Use .. instead of * since we need nested numbering instead of nested bullets. These seem to be sequential steps.